clusters/hive/pools/serverless/serverless-ocp-4-15-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: serverless-ci
+    product: ocp
+    region: us-east-1
+    version: "4.15"
+    version_lower: 4.15.0-0
+    version_upper: 4.16.0-0
+  name: serverless-ocp-4-15-amd64-aws-us-east-1
+  namespace: serverless-cluster-pool
+spec:
+  baseDomain: serverless.devcluster.openshift.com
+  hibernationConfig:
+    resumeTimeout: 20m0s
+  imageSetRef:
+    name: ocp-release-4.15.3-x86-64-for-4.15.0-0-to-4.16.0-0
+  installAttemptsLimit: 1
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  labels:
+    tp.openshift.io/owner: serverless-ci
+  maxSize: 10
```

Comment:
Maybe we should have here some multiple of the number of jobs that usually run per PR. You were saying 4 is the usual number (thus the size), so I think `12` or `8` seams like a better value.

Reply:
We could set e.g. 12. Some repositories have only two parallel checks (e.g. Serving/Eventing), and some have 4 (s-o, EKB).

Reply:
I've set this to 12